zerver/openapi/zulip.yaml

```diff
@@ -8768,21 +8769,21 @@ paths:
           description: |
             Controls which visibility policy to set.
 
-            - 0 - INHERIT
+            - 0 - NONE
             - 1 - MUTED
             - 2 - UNMUTED
```

Comment:
Can we fix all of these all caps in this parameter description to only have the first letter capitalized here, and revise the explanatory text so that they are lowercase and link to the help center ... maybe something like,

> - 0 = None. Removes the visibility policy previously set for the topic.
> - 1 = Muted. [Mutes the topic](/help/mute-a-topic) in a stream.
> - 2 = Unmuted. [Unmutes the topic](/help/mute-a-topic) in a muted stream.
>
> In an unmuted stream, a topic visibility policy of unmuted will have the same effect as the "None" visibility policy.

---

In general, it's good to avoid describing the UI of settings in the API documentation if possible, because it might vary for different clients. For example, the mobile or terminal apps might do something different than the web app to indicate muted topics or streams. That's why linking to the help center documentation can be helpful because there we do try to explain what users will experience in regards to the UI for the web app and also as in this case for the mobile app.